enzyme/Enzyme/GradientUtils.cpp

```diff
@@ -2538,7 +2538,17 @@ Value *GradientUtils::invertPointerM(Value *const oval, IRBuilder<> &BuilderM,
   if (isConstantValue(oval)) {
     // NOTE, this is legal and the correct resolution, however, our activity
     // analysis honeypot no longer exists
-    return lookupM(getNewFromOriginal(oval), BuilderM);
+
+    switch (mode) {
```

Comment:
I think the better solution here would be to have invertPointerM not also call lookup, but any code outside that calls invertpointerM that requires it explicitly call lookup. Thus invertpointer wouldn't need this.